e2e/e2e.go

```diff
@@ -319,4 +343,30 @@ func main() {
 	if err != nil {
 		log.Fatal().Err(err).Msg("E2E tests failed, or failed to run")
 	}
+
+	if killLocalizer {
+		log.Info().Msg("Killing underlying localizer used for devenv tunnel")
+
+		var buf bytes.Buffer
+		cmd = exec.CommandContext(ctx, "pgrep", "localizer")
+		cmd.Stderr = &buf
+		cmd.Stdout = &buf
+		cmd.Stdin = os.Stdin
+		err = cmd.Start()
+		if err != nil {
+			log.Warn().Err(err).Msg("Failed to kill localizer, couldn't find process ID")
+			return
+		}
+
+		fmt.Println("captured PID", buf.String())
+
+		cmd = exec.CommandContext(ctx, "sudo", "kill", strings.TrimSpace(buf.String()))
+		cmd.Stderr = os.Stderr
+		cmd.Stdout = os.Stdout
+		cmd.Stdin = os.Stdin
+		err = cmd.Start()
+		if err != nil {
+			log.Warn().Err(err).Msg("Failed to kill localizer using process ID")
+		}
+	}
```

Comment:
We should probably move this whole thing to some helper function rather than inline.  I doubt this is the last time we want to kill a proc by name.

Reply:
So I ended up having to go a completely different direction because it was proving to be impossible to get the PID of that process using shell commands. I ended up implementing a kill RPC in the localizer server. This seemed like overall a better option for not only this, but also for devenv tunnel. We'll have more control to clean up users potentially messed up environments.